guides/source/active_record_migrations.md

```diff
@@ -931,43 +1183,66 @@ number associated with the previous migration you can run:
 $ bin/rails db:rollback
 ```
 
-This will rollback the latest migration, either by reverting the `change`
-method or by running the `down` method. If you need to undo
-several migrations you can provide a `STEP` parameter:
+This will rollback the latest migration, either by reverting the `change` method
+or by running the `down` method. If you need to undo several migrations you can
+provide a `STEP` parameter:
 
 ```bash
 $ bin/rails db:rollback STEP=3
 ```
 
 The last 3 migrations will be reverted.
 
-The `db:migrate:redo` command is a shortcut for doing a rollback and then migrating
-back up again. As with the `db:rollback` command, you can use the `STEP` parameter
-if you need to go more than one version back, for example:
+In some cases where you modify a local migration and would like to rollback that
+specific migration before migrating back up again, you can use the
+`db:migrate:redo` command. As with the `db:rollback` command, you can use the
+`STEP` parameter if you need to go more than one version back, for example:
 
 ```bash
 $ bin/rails db:migrate:redo STEP=3
 ```
 
-Neither of these rails commands do anything you could not do with `db:migrate`. They
-are there for convenience, since you do not need to explicitly specify the
-version to migrate to.
+NOTE: You could get the same result using `db:migrate`. However, these are there
+for convenience so that you do not need to explicitly specify the version to
+migrate to.
 
-### Setup the Database
+#### Transactions
 
-The `bin/rails db:setup` command will create the database, load the schema, and initialize
-it with the seed data.
+In databases that support transactions with statements that change the schema,
+each migration is wrapped in a transaction.
```

Comment:
"transactions with statements that change the schema" refers to DDL transactions (or transaction DDL).
I think it would help if we mention the term here first.
```suggestion
In databases that support DDL transactions — changing the schema in a single transaction —
each migration is wrapped in a transaction.
```